rustfmt.toml

```diff
@@ -0,0 +1 @@
+hard_tabs = false
```

Comment:
Please remove this file, we use the default `rustfmt` config 🙂

Reply:
Sure thing! I'll keep it local.

I have `hard_tabs = true` in my `~/.config/rustfmt/rustfmt.toml`, so my local rustfmt kept trying to convert all of this project's spaces to tabs :sweat_smile: 

I figured I'd include it so that it didn't happen to other contributors in the future. But I can certainly remove it!

Would you mind if I added `rustfmt.toml` to the `.gitignore` instead? That'll allow me to keep this in the repo so that my local rustfmt doesn't try and convert this repo from spaces to tabs :sweat_smile: